kratos/utilities/model_part_graph_utilities.h

```diff
@@ -0,0 +1,480 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:   Riccardo Rossi
+//
+
+#if !defined(KRATOS_MODELPART_GRAPH_UTILITIES_H_INCLUDED)
```

Comment:
Use pragma once

Reply:
i prefer to keep this one

Reply:
never mind, i changed as i don't feel like discussing about this detail, nevertheless i see perfectly admissible to go on defining as we always did (which btw is how is in the template I used_)